routes/users/users.js

```diff
@@ -5,6 +5,19 @@ const pool = require('../../db');
 const userRouter = express();
 
 userRouter.use(express.json());
+
+// Get user by id endpoint
+userRouter.get('/users/:id', async (req, res) => {
+  try {
+    const { userId } = req.params;
```

Comment:
the param variable name (currently 'userId') _must_ match the param name used in the url ('id')